README.md

```diff
@@ -60,6 +60,25 @@ See the [issue tracker] if you're unsure where to start, or feel free to reach o
 
 See also: the Kubernetes [community page].
 
+## Why kind?
+
+ - kind supports building official Kuberbetes release builds from source
```

Comment:
"official" is probably actually misleading too, these are not blessed by the release team, they just mimic those as much as possible. :thinking: 

s/Kuberbetes/Kubernetes/